Makefile

```diff
@@ -113,7 +113,7 @@ vet: ## Run go vet against code.
 
 .PHONY: test
 test: manifests generate fmt vet envtest ## Run tests.
-	KUBEBUILDER_ASSETS="$(shell $(ENVTEST) use $(ENVTEST_K8S_VERSION) -p path)" go test ./... -coverprofile cover.out
+	KUBEBUILDER_ASSETS="$(shell $(ENVTEST) use $(ENVTEST_K8S_VERSION) --bin-dir $(LOCALBIN) -p path)" go test ./... -coverprofile cover.out
```

Comment:
Is this just a reconciliation with the current scaffold?

Reply:
Yes, just following the upgrade docs.

Reply:
Where is LOCALBIN defined? I don't see that in the Makefile

Reply:
It's been defined [here](https://github.com/redhat-openshift-ecosystem/operator-certification-operator/blob/main/Makefile#L162-L164) since the initial commit I think.